app/src/main/java/com/hedvig/app/ui/view/HedvigEditText.kt

```diff
@@ -54,6 +55,10 @@ class HedvigEditText @JvmOverloads constructor(
         }
     }
 
+    fun getText() = textInput.text.toString()
+
+    fun getEditText(): EditText = textInput
```

Comment:
I think this implementation detail should be hidden by `HedvigEditText` - we could provide a way to add an `onChange`-listener to the parent `HedvigEditText` and pass it down to the wrapped `EditText`.